vol_test.h

```diff
@@ -207,6 +209,18 @@ extern char vol_test_filename[];
 
 #define VOL_TEST_FILENAME_MAX_LENGTH 1024
 
+#define VOL_TEST_PATH_MAX_LENGTH 1024
+
+/* Globally declare buffer name to let macro be used multiple times in same scope */
+extern char prefixed_filename_buffer[0];
+
+/* Prepend prefix to filename in dynamically allocated buffer, use user-declared pointer to return result. */
+#define PREFIX_FILENAME(buf, prefix, filename) \
+prefixed_filename_buffer[strlen(prefix) + strlen(filename) + 1]; \
```

Comment:
I'm not entirely certain how this is working since this statement shouldn't do anything, but I'm assuming it's because the different compilers end up declaring `prefixed_filename_buffer` as a `strlen(prefix) + strlen(filename) + 1`-sized buffer when they see this, as opposed to the size 0 array it was declared as above.

What I meant by dynamically-allocated was that the buffers should be allocated with `malloc` and freed later on with `free`.